src/TLuaInterpreter.cpp

```diff
@@ -9865,6 +9865,14 @@ int TLuaInterpreter::appendCmdLine( lua_State * L )
     return 0;
 }
 
+int TLuaInterpreter::getCmdLine( lua_State * L )
+{
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    QString curText = pHost->mpConsole->mpCommandLine->toPlainText();
+    lua_pushstring( L, curText.toLatin1().data() );
```

Comment:
I have not had a chance to test this, but wouldn't this stuff up text in non-ASCII? See @SlySven's recent use of something else for returning data that's international-compatible.